src/main/java/calculator/Application.java

```diff
@@ -1,7 +1,12 @@
 package calculator;
 
+import static camp.nextstep.edu.missionutils.Console.readLine;
+
 public class Application {
     public static void main(String[] args) {
-        // TODO: 프로그램 구현
+        System.out.println("덧셈할 문자열을 입력해주세요.");
+        String input = readLine();
+        CalculatorService calculatorService = new CalculatorService(input);
+        calculatorService.calculate();
     }
```

Comment:
Application의 역할과 책임에 대해서 단순히 프로그램을 돌린다의 역할을 부여하도록 설계하는게 어떨까 싶습니다.
```suggestion
public class Application {

    public static void main(String[] args) {
        CalculatorController calculatorController = new CalculatorController();
        calculatorController.startCalculating();
    }
}
    }
```
그래서, 프로그램을 시작하는 Application, 계산기를 돌리는 controller 패키지, controller 안에서 출력을 하는 view 패키지로 책임을 분산하도록 설계를 하게 되면 장점은 추후 요구사항이 변경됐을 때 유지보수가 좀 더 쉽지 않을까 싶어요.

Reply:
패턴을 체계가 안 잡힌 상태에서 Application의 역할이 이걸로 괜찮나? 라는 의문을 품었는데,
답변해주셔서 감사합니다!
2주차부터는 controller와 view를 적극적으로 써보겠습니다~!